pkg/gui/keybindings.go

```diff
@@ -1148,6 +1148,14 @@ func (gui *Gui) GetInitialKeybindings() []*Binding {
 			Modifier: gocui.ModNone,
 			Handler:  gui.handleMouseScrollDown,
 		},
+		{
+			ViewName:    "main",
+			Contexts:    []string{"patch-building", "staging"},
```

Comment:
I would keep this to just the staging context for now, because it's not likely that you'll do something in the patch-building context and then want to commit.

I would also add another keybinding for committing in an editor (using `Key:         gui.getKey("files.commitChangesWithEditor"),` and also without a hook using `gui.getKey("files.commitChangesWithoutHook")`